tests/requirements-libraries.txt

```diff
@@ -88,7 +88,7 @@ pyqt5==5.13.2; sys_platform == 'win32'
 pyqt5==5.14.1; sys_platform != 'win32'
 # On Appveyor, newer versions fail with the error message
 # ``[1916:2732:0204/144922.809:ERROR:mf_helpers.cc(14)] Error in dxva_video_decode_accelerator_win.cc on line 513``
-pyqtwebengine==5.12.1; sys_platform == 'win32'
+pyqtwebengine==5.14.0; sys_platform == 'win32'
```

Comment:
This should be `pyqtwebengine==5.12.1; sys_platform == 'win32'  # pyup: ignore`. I'm hoping that at some point someone will figure out why this is happening, or a newer version of the package will fix this. @Legorooj, any ideas?

Reply:
I'll add the comments - in a new PR from my account. It'll be done today. (I just realised I can cherry-pick some of these commits! Git rocks)

Reply:
Done in new PR

Reply:
Thanks!